libs/portals/admin/service-desk/src/screens/Users/Users.tsx

```diff
@@ -0,0 +1,135 @@
+import { Form, useActionData, useNavigate } from 'react-router-dom'
+
+import {
+  AsyncSearchInput,
+  Box,
+  Button,
+  Stack,
+  toast,
+} from '@island.is/island-ui/core'
+import { formatNationalId, IntroHeader } from '@island.is/portals/core'
+import { maskString } from '@island.is/shared/utils'
+import { useLocale } from '@island.is/localization'
+import { useAuth } from '@island.is/auth/react'
+import { replaceParams, useSubmitting } from '@island.is/react-spa/shared'
+
+import * as styles from '../Companies/Companies.css'
+import { useEffect, useState } from 'react'
+import { m } from '../../lib/messages'
+import { Card } from '../../components/Card'
+import { ServiceDeskPaths } from '../../lib/paths'
+import { GetUserProfilesResult } from './Users.action'
+
+const Users = () => {
+  const [focused, setFocused] = useState(false)
+  const [searchInput, setSearchInput] = useState('')
+  const actionData = useActionData() as GetUserProfilesResult
+  const { formatMessage } = useLocale()
+  const navigate = useNavigate()
+  const userProfile = useAuth()
```

Comment:
This is a bit confusing naming when reading markup below.

```suggestion
  const { userInfo } = useAuth()
```